src/repos/ArticleRepo.ts

```diff
@@ -5,24 +5,34 @@ import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES } from '../common/cons
 import { PublicationModel } from '../entities/Publication';
 
 const getArticleByID = async (id: string): Promise<Article> => {
-  return ArticleModel.findById(new ObjectId(id));
+  const article = ArticleModel.findById(new ObjectId(id));
+  if((await article).filtered){
```

Comment:
do the await on the previous line

Reply:
What do you mean by previous line

Reply:
I changed it to be more consistent with other filters